src/Bridge/Symfony/Bundle/Resources/config/swagger.xml

```diff
@@ -16,11 +16,10 @@
             <argument type="service" id="api_platform.operation_method_resolver" />
             <argument type="service" id="api_platform.router" />
             <argument type="service" id="api_platform.iri_converter" />
+            <argument>%api_platform.formats%</argument>
             <argument>%api_platform.title%</argument>
             <argument>%api_platform.description%</argument>
             <argument>2.0</argument>
```

Comment:
Shouldn't be hardcoded. A new optional `version` configuration parameter should be added.